src/components/calcite-tile/calcite-tile.scss

```diff
@@ -0,0 +1,61 @@
+:host-context([theme="dark"]) {
+  @include calcite-theme-dark();
+}
+$spacing: $baseline/2;
+
+:host {
+  background-color: var(--calcite-ui-foreground-1);
+  transition: $transition;
+  color: var(--calcite-ui-text-3);
+  box-sizing: border-box;
+  user-select: none;
+}
+:host(:not([embed])) {
+  box-shadow: 0 0 0 1px var(--calcite-ui-border-2);
+  max-width: 300px;
+  padding: $spacing;
+}
+:host(:not([embed])[href]:hover) {
+  box-shadow: 0 0 0 2px var(--calcite-ui-blue-1);
+  cursor: pointer;
+}
+:host(:not([embed])[href]:active) {
+  box-shadow: 0 0 0 3px var(--calcite-ui-blue-1);
+}
+:host([active]),
+:host(:not([embed])[href]:active) {
+  #heading {
+    font-weight: 500;
+  }
+}
+:host([icon][heading]:not([description]):not([embed])) {
+  padding: unset;
+}
+#tile {
```

Comment:
This is mostly just a pedantic personal thing, but I would prefer not to style on id selectors. Is there a reason you did that?

Reply:
https://github.com/Esri/calcite-components/tree/master/conventions#css-class-names

Reply:
I did it just because I could because of shadow dom.  Forgot that the convention was to use classnames.  I'll fix that up.

Reply:
Done 👍